environments/php7/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM php:7.1-cli
```

Comment:
Would the php7.1-alpine image work?  If all else is equal it would be better to use a smaller image.